tests/serializers/test_union.py

```diff
@@ -510,3 +512,87 @@ class Item(BaseModel):
     )
 
     assert s.to_python([DBUser(name='John', password='secret')]) == [{'name': 'John'}]
+
+
+EXAMPLE_UUID = uuid.uuid4()
+
+
+@pytest.mark.parametrize('order', ['direct', 'inverse'])
```

Comment:
I have a slight preference for 

```suggestion
@pytest.mark.parametrize('reversed', [False, True])
```

but I'm not going to mind if you have a strong preference for `order`.